nixie/unittests/test_verbose.py

```
@@ -0,0 +1,77 @@
+
+
+def test_parse_args_verbose(monkeypatch) -> None:
+    monkeypatch.setattr(sys, "argv", ["nixie", "--verbose", "file.md"])
+    parsed = parse_args()
+    assert parsed.verbose is True
+    assert parsed.paths == [Path("file.md")]
+
+
+@pytest.mark.asyncio
+async def test_render_block_emits_command(
+    monkeypatch, tmp_path: Path, caplog: pytest.LogCaptureFixture
```

Ruff (0.12.2)

Missing type annotation for function argument `monkeypatch`

(ANN001)